group_vars/all.yml

```diff
@@ -81,15 +81,14 @@ dhcp_end: 192.168.24.105
 inspection_iprange: 192.168.24.110,192.168.24.250
 # external network params for adding external network to
 # undercloud to access overcloud resources
-external_gateway: 172.17.5.1/24
-external_network_broadcast: 172.17.5.255
+external_gateway: 172.18.0.1/16
 external_network_vlan_id: 300
-clean_debug: False
+clean_debug: True
```

Comment:
Why this change?

Reply:
This will clean the HDD when we delete the overcloud.
This will avoid booting the overcloud node from the unwanted disk when it has multiple disks.

Reply:
Ack

Reply:
Lets have this as a separate change as it is impacting node intropsection.